mod-source-record-manager-server/src/main/resources/templates/db_scripts/create_get_record_processing_log_function.sql

```diff
@@ -0,0 +1,99 @@
+AS $$
+BEGIN
+    RETURN QUERY EXECUTE format('
+        SELECT records_actions.job_execution_id,
+               records_actions.source_id,
+               records_actions.source_record_order,
+               rec_titles.title,
+               CASE WHEN marc_errors_number != 0 OR marc_actions[array_length(marc_actions, 1)] = ''NON_MATCH''
+                        THEN ''DISCARDED''
+                    WHEN marc_actions[array_length(marc_actions, 1)] = ''CREATE''
+                        THEN ''CREATED''
+                    WHEN marc_actions[array_length(marc_actions, 1)] IN (''UPDATE'', ''MODIFY'')
+                        THEN ''UPDATED''
+               END AS source_record_action_status,
+			   source_entity_id[1]::uuid,
+			   source_entity_hrid[1],
+			   source_entity_error[1],
+               get_entity_status(instance_actions, instance_errors_number) AS instance_action_status,
+			   instance_entity_id[1]::uuid,
+			   instance_entity_hrid[1],
+			   instance_entity_error[1],
+               get_entity_status(holdings_actions, holdings_errors_number) AS holdings_action_status,
+			   holdings_entity_id[1]::uuid,
+			   holdings_entity_hrid[1],
+			   holdings_entity_error[1],
+               get_entity_status(item_actions, item_errors_number)         AS item_action_status,
+			   item_entity_id[1]::uuid,
+			   item_entity_hrid[1],
+			   item_entity_error[1],
+               get_entity_status(order_actions, order_errors_number)       AS order_action_status,
+			   order_entity_id[1]::uuid,
+			   order_entity_hrid[1],
+			   order_entity_error[1],
+               get_entity_status(invoice_actions, invoice_errors_number)   AS invoice_action_status,
+			   invoice_entity_id[1]::uuid,
+			   invoice_entity_hrid[1],
+			   invoice_entity_error[1]
+        FROM (
+               SELECT journal_records.source_id,
+                      journal_records.source_record_order,
+                      journal_records.job_execution_id,
+                      array_agg(action_type ORDER BY action_date ASC) FILTER (WHERE entity_type = ''MARC_BIBLIOGRAPHIC'') AS marc_actions,
+                      count(journal_records.source_id) FILTER (WHERE entity_type = ''MARC_BIBLIOGRAPHIC'' AND journal_records.error != '''') AS marc_errors_number,
+
+			          array_agg(entity_hrid) FILTER (WHERE entity_type = ''MARC_BIBLIOGRAPHIC'') AS source_entity_hrid,
+			          array_agg(entity_id) FILTER (WHERE entity_type = ''MARC_BIBLIOGRAPHIC'') AS source_entity_id,
```

Comment:
let's remove this, looks like we don't use it to fill the DTO

Reply:
@RuslanLavrov As far as  I understand, we have this field in the schema: "sourceRecordId". Am I wrong?

Reply:
@VRohach, yes agree. As I see we use "source_id" column to fill "sourceRecordId" property.  Since we have record id in "source_id" column I propose to remove this calculation.

Reply:
@RuslanLavrov Ok. Got it! Thank you